Makefile

```diff
@@ -98,7 +98,7 @@ Dockerfile.rhel7: Dockerfile Makefile
 
 # This was copied from https://github.com/openshift/cluster-image-registry-operator
 test-e2e:
-	go test -tags=$(GOTAGS) -failfast -timeout 90m -v$${WHAT:+ -run="$$WHAT"} ./test/e2e/
+	go test -tags=$(GOTAGS) -failfast -timeout 100m -v$${WHAT:+ -run="$$WHAT"} ./test/e2e/
```

Comment:
can we add these tests as a new e2e like we did for single node below? we'll want to test more and there isn't a reasonable way to add them to this tests suite as it's already 90m. We previously spent a lot of time getting these tests down to around 90 and we shouldn't let them go higher. It's easier to do this now as opposed to later.

Reply:
I wholeheartedly agree with this statement. If we do that, we'll also need a PR to https://github.com/openshift/release/blob/master/ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml to add another test step target.

Reply:
I'm not even really sure the test that required that bump should get merged to master. It's a PoC, not actually testing MCO code

Reply:
That's a good point. For now, we can probably remove that commit from this PR. I need to find a new location for it though.